### PR TITLE
migration for sample-type

### PR DESCRIPTION
**Pull request by sayo96825**
Separate PR to make migration to just add sample type as we discussed.

### CODE REVIEW

File: LIMS_IMAGE/web/laboratory/models.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def __str__(self):
         return self.name # tests are referenced by their name
     # By default, Django gives each model an auto-incrementing primary key with the type specified per app
     name = models.CharField(max_length=100) # name of the test
+    sample_type = models.CharField(max_length=100, null=True)
```
Comment (shepkeira) [inline LIMS_IMAGE/web/laboratory/models.py:29]: please add a comment here explaining what this attribute is used for

**Review by shepkeira — CHANGES_REQUESTED** [review]
You will need to
1) fill in the sample_type for the tests we already have, look at connected samples types, for what the sample_type should be for that test. Or just take an educated guess
2) You need to backup the database. This is why we needed it as a separate PR. Do this after you have updated the database, and filled in the sample_types for the different tests we already have

File: LIMS_IMAGE/web/laboratory/models.py (hunk 1 above)
Comment (sayo96825) [reply LIMS_IMAGE/web/laboratory/models.py:29]: So I can go to the admin page and add those sample type and save it now from the sample-type branch. 
Then I can backup the database.

Comment (shepkeira) [reply LIMS_IMAGE/web/laboratory/models.py:29]: yes

Comment (sayo96825) [reply LIMS_IMAGE/web/laboratory/models.py:29]: Just to confirm, sample types are entered into the database in the admin page.  
image

So now I can backup the database.

**Review by shepkeira — APPROVED** [review]
Looks good now

**Pull request merged**